unixExample/backends/ImGuiNotify.hpp

```diff
@@ -0,0 +1,648 @@
+/**
+ * @file ImGuiNotify.hpp
+ * @brief A header-only library for creating toast notifications with ImGui.
+ * 
+ * Based on imgui-notify by patrickcjk
+ * https://github.com/patrickcjk/imgui-notify
+ * 
+ * @version 0.0.1 by TyomaVader
+ * @date 28.07.2023
+ */
+
+#ifndef IMGUI_NOTIFY
+#define IMGUI_NOTIFY
+
+#pragma once
+
+#include <vector>			// Vector for storing notifications list
+#include <string>
+#include <chrono>			// For the notifications timed dissmiss
+#include <functional>		// For storing the code, which executest on the button click in the notification
+
+#include "imgui.h"
+#include "imgui_internal.h"
+
+#include "IconsFontAwesome6.h"
+#include "fa_solid_900.h"
```

Comment:
windows cmake  fail with this header `fa_solid_900.h`, suggest removing it. No use according to me.